ajax/libs/emojione/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "emojione",
   "title": "emojione",
   "filename": "lib/js/emojione.min.js",
-  "version": "2.2.7",
+  "version": "3.0.1",
   "description": "Emoji One is a complete set of emojis designed for the web. It includes libraries to easily convert unicode characters to shortnames (:smile:) and shortnames to our custom emoji images. PNG and SVG formats provided for the emoji images.",
   "homepage": "http://www.emojione.com",
```

Comment:
please use another commit to update the homepage "https://www.emojione.com/"
let's use https